docs/src/topics/flavors/vpcless.md

```diff
@@ -27,3 +30,17 @@ supported in this flavor and it uses VXLAN for pod to pod communication.
     ```bash
     kubectl apply -f test-cluster.yaml
     ```
+
+### Flatcar
```

Comment:
I think the flatcar flavor is unique and important enough that we should add it as it's own page, it will make it easier to search for and make it more prominent that it is supported

Reply:
My immediate thought was to add it to the flavors folder, but we haven't been super disciplined about where we add docs so I would be fine as either a flavor doc or a topic doc